src/main/java/org/folio/circulation/CirculationVerticle.java

```diff
@@ -52,13 +53,18 @@ public void start(Future<Void> startFuture) {
     new CheckOutByBarcodeResource("/circulation/override-check-out-by-barcode",
       client, new OverrideCheckOutStrategy()).register(router);
     new CheckInByBarcodeResource(client).register(router);
-    new RenewByBarcodeResource(client).register(router);
-    new RenewByIdResource(client).register(router);
+
+    new RenewByBarcodeResource("/circulation/renew-by-barcode",
```

Comment:
Can you help me understand these changes? Is it only to avoid the Sonar warning about hard coded paths or URLs?

Reply:
It's the intent to reuse RenewByBarcodeResource for both regular renewal and renewal override by passing different renewal strategies.
 It was done, as number of logic for renewal grows, and it's almost the same for renewal and override. I did it in the same way as for CheckOutByBarcodeResource.

Reply:
Thanks for the explanation. I noticed `RenewByIdResource` is only used once (I think) and yet has path and strategy parameters. I can understand the value in keeping that consistent.